### DISCUSSION TITLE
HUD Controls "Text Box"

### DISCUSSION THREAD

**Discussion by Warren-Whitfield** [pelicanmapping/osgearth#2552] [Support Q&A] [answered]
Is there a control similar to a LabelControl, which is editable?
I would like to add editable text to this sort of HUD.

![image](https://github.com/gwaldron/osgearth/assets/159110675/fbbe1242-6df0-4524-8db9-90acf00aaa66)

**Comment by gwaldron** [comment, marked as answer]
No, sorry. You might want to investigate ImGui for HUD controls.
